### DISCUSSION TITLE
Dynamically updating react Cropper image not shows image

### DISCUSSION THREAD

**Discussion by kumargaurav** [ValentinH/react-easy-crop#449] [Q&A]
I have used Cropper and loading the image from React state. but unfortunately when the state updates the image is not showing up in cropper until any user interaction done.

<img width="597" alt="image" src="https://user-images.githubusercontent.com/1484621/221869813-e5165e6f-22b4-44bb-835c-b2cb1ea990af.png">

After click  or drag on the cropper area it shows up the image as follows

<img width="552" alt="image" src="https://user-images.githubusercontent.com/1484621/221870162-02fd44e9-c64e-4faa-993e-7943923fdc84.png">

Initially the image source will be empty. once user upload image then we are updating the react state.
    <Cropper
                                    image={this.state.uploadedbase64}
                                    crop={this.state.crop}
                                    zoom={this.state.zoom}
                                    aspect={this.state.aspect}
                                    onCropChange={this.onCropChange}
                                    onCropComplete={this.onCropComplete}
                                    onZoomChange={this.onZoomChange}
                                />

Please let me know what am i missing here

**Comment by ValentinH** [comment]
In order to effectively help you, it would be great to create a codesandbox showing the issue.

**Reply by kumargaurav** [reply]
changing zoom value from 0 to 1 resolved the issue.
